bin/verify_release_assets

```diff
@@ -0,0 +1,126 @@
+#!/bin/sh
+
+set -e
+
+. libstd-ecm.sh
+
+: "${REPOSITORY:=rancher/rke2}"
+: "${STRICT_MATCH:=false}"
+: "${LIST_ASSETS:=false}"
+: "${ASSERT_AMOUNT:=}"
+
+usage() {
+    echo "usage: $0 [adhlr] <repository>
+    -l    list assets file names
+    -r    set target repository. default: $REPOSITORY
+    -s    use strict release tag name match
+    -a    override expected assets amount
+    -h    show help
+
+examples:
+    $0 -s v1.23.4
+    $0 -s v1.23.4-rc2+rke2r1
+    $0 -l -r rancher/rke2 v1.23.4"
+}
+
+print_status() {
+    release=${1}
+    current_count=${2}
+    expected=''
+
+    case $REPOSITORY in
```

Comment:
Can you add `{` and `}` to the variables in this script for consistency with the rest of the scripts?